src/app/app-routing.module.ts

```diff
@@ -2,6 +2,7 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { CorePath } from '@app/shared/models';
 import { MyAccountComponent } from './my-account/my-account/my-account.component';
+import { BreadcrumbLabels } from '@app/shared/models/breadcrumb-labels';
```

Comment:
Just:
```suggestion
import { BreadcrumbLabels } from '@app/shared/models';
```
If there's no such export, re-export it from `index.ts` of the `models` folder.